<launch>
  <arg name="port" default="/dev/ttyUSB0"/>
  <arg name="baud_rate" default="115200"/>
  <arg name="slave_id" default="9"/>
  <arg name="publish_rate" default="25.0"/>
  <arg name="default_speed" default="0.3"/>
  <arg name="default_force" default="0.5"/>
  <arg name="auto_activate" default="true"/>
  <arg name="position_units" default="normalized"/>

  <node pkg="gripper_ros" type="robotiq_node" name="robotiq_node" output="screen">
    <param name="port" value="$(arg port)"/>
    <param name="baud_rate" value="$(arg baud_rate)"/>
    <param name="slave_id" value="$(arg slave_id)"/>
    <param name="publish_rate" value="$(arg publish_rate)"/>
    <param name="default_speed" value="$(arg default_speed)"/>
    <param name="default_force" value="$(arg default_force)"/>
    <param name="auto_activate" value="$(arg auto_activate)"/>
    <param name="position_units" value="$(arg position_units)"/>
  </node>
</launch>
